keps/sig-cluster-lifecycle/kubeadm/20190722-Advanced-configurations-with-kubeadm-(Kustomize).md

```diff
@@ -209,7 +202,17 @@ metadata:
 EOF
 ```
 
-2. Run kubeadm passing the new patch, e.g.
+In case higher-level tools/users are providing only strategic merge patches, like in the example above,
+it is not requested to take care of defining `kustomization.yaml` file; in case the `kustomization.yaml` 
```

Comment:
`of defining` -> `of defining a`
`; in case the `kustomization.yaml` -> `. In case the `kustomization.yaml` file`